apps/src/blockly/cdoBlocklyWrapper.js

```diff
@@ -275,6 +275,13 @@ function initializeBlocklyWrapper(blocklyInstance) {
     getCode: function (workspace) {
       return Blockly.Xml.domToText(Blockly.Xml.blockSpaceToDom(workspace));
     },
+    playSoundButton: function (dashboard, onSelect) {
```

Comment:
Nit: What if we called this `playSoundField` (or `soundField`)? While the new field will likely work more like a button, this version is still technically a dropdown.